### DISCUSSION TITLE
When we will get the latest release ?

### DISCUSSION THREAD

**Discussion by charlesk40** [jwtk/jjwt#813] [Q&A] [answered]
Hi there, I would like to get the latest (master) changes which includes JWE, JWK etc.  When will we expect to see the latest changes released?  Thanks!

**Comment by lhazlewood** [comment, marked as answer]
Hi @charlesk40 !

As an unpaid volunteer team, we cannot make any date or timeline promises.

That said, we're actively working on resolving all necessary [Issues](https://github.com/jwtk/jjwt/issues) to cut the `0.12.0` release as soon as possible.  Last week was a _huge_ time sink due to all the effort in testing PKCS11 keys on a (simulated) Hardware Security Module (HSM).  But that is finished and fully automated in CI now, so this and next week is all about finishing the outstanding issues that are required for full JWE/JWK functionality.  I hope(!) we can get the release out by the end of next week or early the week thereafter. 🤞

**Reply by hertg** [reply]
I see in the commits that you've been working hard on the project lately, I'm very much looking forward to the next release.
Have you thought about setting up Github Sponsors, or something similar? If you set something up feel free to ping me, I'll forward the link at my workplace. I want to make sure we toss a few coins at FOSS developers who make things we find useful :)

**Reply by lhazlewood** [reply]
Hi @hertg  yes, we've been working super diligently, and thanks for the suggestion!  We haven't discussed that  option before, but it might be something to look into. We can definitely discuss after the release 😁.

**Comment by lhazlewood** [comment]
`0.12.0` has been released! https://github.com/jwtk/jjwt/releases/tag/0.12.0
